queries.sql

```diff
@@ -60,3 +60,39 @@ SELECT AVG(escape_attempts), species
 FROM animals
 WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
 GROUP by species;
+
+-- Answer join questions
+SELECT *
+FROM owners JOIN animals
+ON owners.id = animals.owner_id
+WHERE owners.full_name = 'Melody Pond';
+
+SELECT *
+FROM species JOIN animals
+ON species.id = animals.species_id
+WHERE species.name = 'Pokemon';
+
+SELECT owners.id, owners.full_name, animals.id, animals.name, animals.owner_id
+FROM owners LEFT JOIN animals
+ON owners.id = animals.owner_id;
+
+SELECT species.name, COUNT(species.name)
+FROM animals JOIN species
+ON species_id = species.id 
+GROUP BY species.id;
+
+SELECT owners.id, owners.full_name, animals.id, animals.name, animals.owner_id
+FROM owners LEFT JOIN animals
+ON owners.id = animals.owner_id
+WHERE owners.full_name = 'Jennifer Orwell';
```

Comment:
- [ ] It would be better if we observe this query carefully to see if it answers the question `List all Digimon owned by Jennifer Orwell`. I don't see the species `Digimon` in the query results